src/test/resources/application.conf

```diff
@@ -15,8 +15,28 @@
 environment: local
 
 local {
+  api-name: ""
+  api-platform-test-user: ""
+  api-platform-agents-auth: ""
+  api-platform-agents-auth-test: ""
+  auth-login-stub_uri: "/government-gateway/session/login"
+  redirect-url: "http://localhost:9949/auth-login-stub/session"
+  tgp-api-uri: ""
+
   services {
     host: "http://localhost"
+    auth {
+      port = 8585
+      productionRoute = ""
+    }
+    ims {
+      port = 9653
+      productionRoute = ""
+    }
+    tgp {
```

Comment:
is this the trader-goods-profiles API?

Reply:
@shiprashipra You can use this ports - https://github.com/hmrc/trader-goods-profiles/blob/main/conf/application.conf

is that correct @mauro-fanton ?

Reply:
Please review it again updated it

Reply:
Thanks. Looks fine to me. @mauro-fanton are you happy with the changes?

Reply:
just would like to know what service id TGP, if this is the trader-goods-profile API then the port number is wrong